boards/posix/nrf5x_bsim/irq_handler.c

```diff
@@ -29,6 +29,42 @@ static struct _isr_list irq_vector_table[NRF_HW_NBR_IRQs];
 
 static int currently_running_irq = -1;
 
+#if defined(CONFIG_SOC_SERIES_NRF53X) || \
+    defined(CONFIG_SOC_COMPATIBLE_NRF53X_CPUNET) || \
+    defined(NRF53_SERIES)
+const char *irqnames[] = { /*just for the traces*/
```

Comment:
You may consider adding the remaining IRQ names as well, see nrf5340_network.h. 
* IPC
* EGU